README.md

```diff
@@ -1,37 +1,44 @@
 # JUMP-single-cell
 
-Single cell analysis of the JUMP Cell Painting consortium data.
-Please ensure adequate storage as the sqlite plate data is approximately 1.1 TB large.
+## Data
 
-## 0.download_data
+In this repository, we apply the [phenotypic profiling model](https://github.com/WayScience/phenotypic_profiling_model), which predicts phenotypic class of single cells using nuclei features, on the [JUMP-Target pilot data](https://github.com/jump-cellpainting/JUMP-Target) from the [JUMP consortium](https://jump-cellpainting.broadinstitute.org/).
 
-This folder contains the code and metadata file to download the plate data from aws.
-For more information on how the plate paths were generated please review the `0.generate_jump_dataset_manifest.ipynb` file.
+In this dataset, there are 51 plates that are associated with one of three treatment types:
 
-To download the data, first run `0.generate_jump_dataset_manifest.ipynb`, to generate the paths data, then run `1.download_jump_single_cell_profiles.ipynb` to download the plate data.
+- Compound
+- CRISPR
+- ORF
 
-## 1.process_data
+Each treatment has it's own platemap and metadata file that can be found in the [reference_plate_data](./reference_plate_data/) folder.
+A barcode platemap is include which associates each plate to the correct platemap file.
 
-In this step, we merged the single cell data using pycytominer, and the reference plate data located in the `reference_plate_data` folder in the root of the git directory.
-The metadata files in this folder can be located by following this aws link:
-`https://cellpainting-gallery.s3.amazonaws.com/index.html#cpg0000-jump-pilot/source_4/workspace/metadata/external_metadata/`
+There are a total of **20,959,860 single cells** between all plates.
 
-The platemaps files in this folder can be found using this aws link:
-`https://cellpainting-gallery.s3.amazonaws.com/index.html#cpg0000-jump-pilot/source_4/workspace/metadata/platemaps/2020_11_04_CPJUMP1/platemap/`
+To reproduce this project, please ensure adequate storage as the CellProfiler SQLite database files is approximately 1.1 TB large.
 
-Similarly the barcode file `barcode_platemap.csv` came from dropbox.
+## Goal
 
-After merging the single cells, each of the features of the plate data were normalized by pycytominer and converted to parquet files.
+Traditionally, image-based analyses aggregate single-cell features which removes outliers that could be interesting biology.
+By predicting phenotypes for single-cells with the phenotypic profiling model, we hope to uncover patterns of biology that wouldn't be uncovered with the traditional methodology.
```

Comment:
```suggestion
By predicting single-cell phenotypes with our phenotypic profiling model, we hope to uncover important patterns of biology that would be missed with the traditional methodology.

Specifically, the benefits of single-cell phenotyping include:

- Granular phenotypic mechanisms of perturbations regarding (A) the impact perturbations have on a specific phenotype (e.g., disrupting mitosis) and (B) impact on phenotype prevalence (e.g., a gene knockout that causes apoptosis or stalls cells in a specific cell cycle phase)
-  Filter and/or combine cells of the same phenotypic class to purify and/or improve the traditional image-based profiling pipeline
- Adding knowledge to specific combinations of morphology features allows for self-referential interpretation, without the need for database signature lookup or other guilt-by-association methods
- When combined with different experimental designs (e.g., targeted fluorescence marker), we can test specific hypotheses regarding single-cell phenotype distributions (and other important hypotheses that would otherwise be impossible without single-cell phenotypes).
```

Reply:
I appreciate this in-depth clarification of the importance of single-cell phenotyping!

I have included this!